packages/trader/src/AppV2/Containers/Trade/trade.tsx

```diff
@@ -130,6 +131,7 @@ const Trade = observer(() => {
             )}
             <ServiceErrorSheet />
             <ClosedMarketMessage />
+            <TradeParamErrorSnackbar trade_params={['stop_loss', 'take_profit', 'date_start']} should_show_snackbar />
```

Comment:
Is it better to keep it here or push it one level down into the TradeParam module?

Reply:
The whole idea was to but it here (like `<ServiceErrorSheet />`), because in this case we have only one instance of this component and as a result, only one snackbar appear. If we put it inside of the TradeParams, then we'll have 2 instances and 2 snackbars will be triggered

Reply:
Maybe I can rename it? Because to be more accurate, this component is handling errors based on passed `error_fields`. So instead of `TradeParamErrorSnackbar` ---> `TradeErrorSnackbar`, `trade_params` --> `error_fields`. What do you think?